src/agent/coverage/src/block.rs

```diff
@@ -95,3 +118,151 @@ impl BlockCov {
         Self { offset, count: 0 }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // Builds a `ModuleCov` from a vec of `(offset, counts)` tuples.
+    fn from_vec(data: Vec<(u64, u32)>) -> ModuleCov {
+        let offsets = data.iter().map(|(o, _)| *o);
+        let mut cov = ModuleCov::new(offsets);
+        for (offset, count) in data {
+            for _ in 0..count {
+                cov.increment(offset);
+            }
+        }
+        cov
+    }
+
+    // Builds a vec of `(count, offset)` tuples from a `ModuleCov`.
+    fn to_vec(cov: &ModuleCov) -> Vec<(u64, u32)> {
+        cov.blocks.iter().map(|(o, b)| (*o, b.count)).collect()
+    }
+
+    #[test]
+    fn test_to_from_vec_roundtrip() {
```

Comment:
Based on the test name and code, it looks like it only tests test code, but I think it's meant to test `increment` so maybe a different function name would be better?

Reply:
Oops, nope, this is just a vestigial self-check test.